src/config.c

```diff
@@ -248,26 +283,34 @@ dictType optionToLineDictType = {
 };
 
 dictType optionSetDictType = {
```

Comment:
This is now unused.

Reply:
This is used in `rewriteConfigCreateState`